.azure-pipelines.yml

```
@@ -91,32 +115,49 @@ jobs:
       export PATH="$HOME/.local/bin:$PATH"
       sudo curl -L -o /usr/local/bin/cmake-easyinstall https://raw.githubusercontent.com/ax3l/cmake-easyinstall/main/cmake-easyinstall
       sudo chmod a+x /usr/local/bin/cmake-easyinstall
-      if [ "${WARPX_CI_OPENPMD:-FALSE}" == "TRUE" ]; then
-        cmake-easyinstall --prefix=/usr/local                   \
-          git+https://github.com/openPMD/openPMD-api.git@0.14.3 \
-          -DCMAKE_CXX_COMPILER_LAUNCHER=$(which ccache)         \
-          -DCMAKE_VERBOSE_MAKEFILE=ON                           \
-          -DopenPMD_USE_PYTHON=OFF -DBUILD_TESTING=OFF -DBUILD_EXAMPLES=OFF -DBUILD_CLI_TOOLS=OFF
-        python3 -m pip install --upgrade openpmd-api
-      fi
-      if [[ "${WARPX_CI_RZ_OR_NOMPI:-FALSE}" == "TRUE" ]]; then
-        cmake-easyinstall --prefix=/usr/local git+https://github.com/icl-utk-edu/blaspp.git \
+      #if [ "${WARPX_CI_OPENPMD:-FALSE}" == "TRUE" ]; then
+      #  cmake-easyinstall --prefix=/usr/local                   \
+      #    git+https://github.com/openPMD/openPMD-api.git@0.14.3 \
+      #    -DCMAKE_CXX_COMPILER_LAUNCHER=$(which ccache)         \
+      #    -DCMAKE_VERBOSE_MAKEFILE=ON                           \
+      #    -DopenPMD_USE_PYTHON=OFF -DBUILD_TESTING=OFF -DBUILD_EXAMPLES=OFF -DBUILD_CLI_TOOLS=OFF
+      #  python3 -m pip install --upgrade openpmd-api
+      #fi
+      if [ "${WARPX_RZ_FFT:-FALSE}" == "TRUE" ]; then
+        # BLAS++
+        cmake-easyinstall --prefix=/usr/local           \
+          git+https://github.com/icl-utk-edu/blaspp.git \
           -DCMAKE_CXX_COMPILER_LAUNCHER=$(which ccache) \
           -DCMAKE_CXX_STANDARD=17                       \
           -Duse_openmp=OFF -Dbuild_tests=OFF -DCMAKE_VERBOSE_MAKEFILE=ON
-        cmake-easyinstall --prefix=/usr/local git+https://github.com/icl-utk-edu/lapackpp.git \
-          -DCMAKE_CXX_COMPILER_LAUNCHER=$(which ccache) \
-          -DCMAKE_CXX_STANDARD=17                       \
+        # LAPACK++
+        cmake-easyinstall --prefix=/usr/local             \
+          git+https://github.com/icl-utk-edu/lapackpp.git \
+          -DCMAKE_CXX_COMPILER_LAUNCHER=$(which ccache)   \
+          -DCMAKE_CXX_STANDARD=17                         \
           -Duse_cmake_find_lapack=ON -Dbuild_tests=OFF -DCMAKE_VERBOSE_MAKEFILE=ON
       fi
+      # Python modules required for test analysis
+      python3 -m pip install --upgrade -r Regression/requirements.txt
```

Clear pip cache (see #5110):
```suggestion
      python3 -m pip install --upgrade -r Regression/requirements.txt
      python3 -m pip cache purge
```